.github/CODEOWNERS

```diff
@@ -202,10 +202,10 @@
 # ServiceOwners:                                                   @ryogok @TFR258 @tburns10 @areddish @toothache
 
 # PRLabel: %Cognitive - Face
-/sdk/cognitiveservices/Vision.Face/                                @JinyuID @dipidoo @SteveMSFT @msyache @longli0 @ShaoAnLin @lulululululu
+/sdk/face/                                                         @JinyuID @dipidoo @longli0 @ShaoAnLin @lulululululu @leareai @Han-msft
```

Comment:
Why are we changing this rather than adding a new entry with path to the new library?

Reply:
I thought we should replace it. If we are going to add new entry, do we also need to copy the service section?

Reply:
If you replace it, then you have no reviewers for the old path.    We can add additional PR entries because they're differentiated by path.   You cannot add a new `ServiceOwners` entry because it is differentiated by label.   Since we're conforming to the existing pattern and reusing the Cognitive label, there can be only one service owners block.